pkg/controllers/leasegarbagecollection/metrics.go

```diff
@@ -0,0 +1,37 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package leasegarbagecollection
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	crmetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+
+	"github.com/aws/karpenter-core/pkg/metrics"
+)
+
+var (
+	NodeLeaseDeletedCounter = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "karpenter",
+			Subsystem: metrics.NodeSubsystem,
+			Name:      "leases_deleted",
+			Help:      "Number of deleted leaked leases. Labeled by owning provisioner.",
```

Comment:
```suggestion
			Help:      "Number of deleted leaked leases.",
```

This isn't labeled by the owning provisioner